tutorials/llm/llama-3/pruning-distillation/introduction.ipynb

```diff
@@ -0,0 +1,190 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "411e6711-60fc-4488-8aa1-c6463cac8695",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Pruning and Distillation of Llama 3.1 model with NeMo Framework"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03fd1cf4-c67a-4b8d-a5e5-46531be0f991",
+   "metadata": {},
+   "source": [
+    "This demonstration showcases performing pruning and distillation on **Llama 3.1-8B** with the [WikiText-103-v1](https://huggingface.co/datasets/Salesforce/wikitext/viewer/wikitext-103-v1) dataset using NeMo Framework. The [WikiText-103-v1](https://huggingface.co/datasets/Salesforce/wikitext/viewer/wikitext-103-v1) language modeling dataset is a collection of over 100 million tokens extracted from the set of verified 'Good' and 'Featured' articles on Wikipedia. \n",
+    "\n",
+    "For this demonstration, we will perform a light finetuning procedure on the `Meta Llama 3.1 8B` teacher model to generate a finetuned teacher model. This finetuned teacher model will then be trimmed. There are two methods to prune a model: depth-pruning and width-pruning. This workflow will showcase both methods which will yield `4b_depth_pruned_model.nemo` and `4b_width_pruned_model.nemo` respectively, that will serve as a starting point for distillation to the final 4B models. \n",
+    "\n",
+    "> We are using models utilizing the `meta-llama/Meta-Llama-3.1-8B` tokenizer for this demonstration.\n",
+    "\n",
+    "> `NOTE:` Ensure that you run this notebook inside the [NeMo Framework container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo) which has all the required dependencies. \n",
+    "\n",
+    "**Instructions are available in the associated tutorial README to download the model and the container.**"
```

Comment:
revise note text and add a link to the README file

  "**Instructions for downloading the model and the container are available in the README.**"